index.html

```diff
@@ -13,7 +13,7 @@
 <link rel="preconnect" href="https://fonts.googleapis.com">
 <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
 <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
-  <title>Document</title>
+  <title>Yacoubou Seidou - Portoflio</title>
```

Comment:
- [ ] Great Job So far but There is something you still need to work on for your website to be accessible, I noticed that some of the text in your code has very low contrast with the background. This can make it difficult for users, especially those with visual impairments, to read the text. Can you please adjust the contrast to ensure that the text is legible and accessible for all users? :wink:  if you wanna [check your contrast use this tool](https://www.aditus.io/button-contrast-checker/)

Reply:
@Alfredbis29  We need to conform with the Figma design so I don't think we can change the text color or background color